src/Emulator/Extensions/Utilities/GDB/Commands/ReadRegisterCommand.cs

```diff
@@ -20,17 +20,34 @@ public ReadRegisterCommand(CommandsManager manager) : base(manager)
         public PacketData Execute(
             [Argument(Encoding = ArgumentAttribute.ArgumentEncoding.HexNumber)]int registerNumber)
         {
-            if(!manager.Cpu.GetRegisters().Any(x => x.Index == registerNumber))
-            {
-                return PacketData.ErrorReply(1);
+            var content = new StringBuilder();
+
+            // if register exists in emulated core return current value of this
+            if(manager.Cpu.GetRegisters().Any(x => x.Index == registerNumber)) { 
```

Comment:
Just a little detail - opening `{` should go to the next line according to our formatting standards